x-pack/plugins/observability_solution/apm/server/lib/helpers/get_apm_alerts_client.ts

```diff
@@ -31,7 +31,13 @@ export async function getApmAlertsClient({
 
   const ruleRegistryPluginStart = await plugins.ruleRegistry.start();
   const alertsClient = await ruleRegistryPluginStart.getRacClientWithRequest(request);
-  const apmAlertsIndices = await alertsClient.getAuthorizedAlertsIndices(['apm']);
+  const apmAlertsIndices = await alertsClient.getAuthorizedAlertsIndices([
```

Comment:
I would try to use `AlertConsumers...` here in order to avoid hardcoding

Reply:
There is an ongoing discussion about this topic in this [comment](https://github.com/elastic/kibana/pull/202025#discussion_r1861847508).